src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,19 @@
+package calculator
+
+object Calculator {
+
+    fun execute(input: String?): Double {
+        require(!input.isNullOrBlank()) { "Input is null or blank." }
```

Comment:
require 활용 👍 👍 아래와 같은 방법도 있겠어요 😄 
```suggestion
    fun execute(input: String?): Double {
        require(input.isNullOrBlank().not()) { "Input is null or blank." }
```